tools/cli/main.js

```diff
@@ -745,6 +745,35 @@ Fiber(function () {
     // It is a plain old argument!
     rawArgs.push(term);
   }
+  
+  // Check if meteor is run as root only on UNIX platforms
+  var unixPlatforms = { 
+    'darwin': true,
+    'linux': true,
+    'freebsd': true
+  };
+  var platform = process.platform;
+
+  if (_.has(unixPlatforms, platform)){
+    var isRoot = process.env.USERNAME == 'root';
```

Comment:
It seems like the canonical way to do this check is [`process.getuid && process.getuid() === 0`](https://github.com/sindresorhus/is-root/blob/86962be252d45e26a78a085e8ad97e7781f2ce34/index.js#L3). The `process.getuid` function is only defined on POSIX platforms, and 0 is the UID of the root user.